integration-tests/pkg/executor/executor_docker_api.go

```diff
@@ -130,6 +130,28 @@ func (d *dockerAPIExecutor) StartContainer(startConfig config.ContainerStartConf
 		return "", errors.Wrapf(err, "start %s", startConfig.Name)
 	}
 
+	tickSeconds := time.Second
+	timeout := 30 * time.Second
+
+	tick := time.Tick(tickSeconds)
+	timer := time.After(timeout)
+
+	for {
```

Comment:
We have a `Retry` method that is currently unused, maybe we can move it to the `common` package and use that here?
https://github.com/stackrox/collector/blob/19eafba2eaa3bc48a3d059ca8f190a22fcc25b09/integration-tests/pkg/executor/retry.go#L19

Reply:
Actually, just noticed the changes are inside the executor package, so it could be used directly if we wanted to.

Reply:
The original Retry was number based, but in this context time based retry is more appropriate. I've moved the ticker implementation into the retry module.